src/makeRobot.js

```diff
@@ -39,6 +39,67 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const SERVICE_CENTER_COORDINATE_X = 1400;
+  const SERVICE_CENTER_COORDINATE_Y = 500;
+
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
```

Comment:
```suggestion
      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
```
If you use interpolation, you don't need to concat strings.

Reply:
> ```suggestion
> 
>       return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
> 
> ```
> 
> If you use interpolation, you don't need to concat strings.

Firstly I had it without concat, but linter says that length of the string is too big, so I had to use concat